### PR TITLE
chore: Add documentation for the new FlagshipUI API

### PR DESCRIPTION
**Pull request by Ldoppea**
This PR documents the new FlagshipUI that has been implemented in #1082

Formated result can be read here: https://github.com/cozy/cozy-flagship-app/blob/551b317fb1533f7d257fed4e5e05861ddd23a911/src/app/view/FlagshipUI/README.md

### CODE REVIEW

File: src/app/view/FlagshipUI/README.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
+# `FlagshipUI` API Documentation
+
+
+## Nomenclature
+
+- `Status bar`: The phone's top bar where notification, clock and network info are displayed
+- `Navigation bar`: The phone's top bar where navigation buttons are displayed
```
Comment (zatteo) [inline src/app/view/FlagshipUI/README.md:7]: It is bottom bar no?

**Comment by zatteo** [comment]
Very nice. Maybe add a link here https://github.com/cozy/cozy-flagship-app/blob/master/docs/README.md ?

**Review by zatteo — APPROVED** [review]

**Comment by zatteo** [comment]
Just a thought : I wonder if it could be a standalone library. A lot of apps have this `StatusBar` mess issues.

File: src/app/view/FlagshipUI/README.md (hunk 1 above)
Comment (Ldoppea) [reply src/app/view/FlagshipUI/README.md]: Good catch. Fixed in https://github.com/cozy/cozy-flagship-app/compare/551b317fb1533f7d257fed4e5e05861ddd23a911..66ebb20b8471d1c760fb597d32e41c85574a5514

**Comment by Ldoppea** [comment]
> Very nice. Maybe add a link here https://github.com/cozy/cozy-flagship-app/blob/master/docs/README.md ?

Good idea, I added it there: https://github.com/cozy/cozy-flagship-app/compare/551b317fb1533f7d257fed4e5e05861ddd23a911..66ebb20b8471d1c760fb597d32e41c85574a5514

**Comment by Ldoppea** [comment]
> Just a thought : I wonder if it could be a standalone library.

It's a possibility yes. This would imply a bit more work to make it generic + the solution based on the ScreenIndex enum may be too simple for complex apps.

> A lot of apps have this `StatusBar` mess issues.

Do you have any examples of apps with this problem?

**Pull request merged**